### DISCUSSION TITLE
Close a finished project

### DISCUSSION THREAD

**Discussion by dang1507** [marcusolsson/obsidian-projects#523] [Help] [answered]
How do I close out a finished project? I only have one project right now and have taken it off the default, but I was curious if there was a way to mark a project as complete?

**Comment by marcusolsson** [comment]
Hi! 

Closing or archiving projects is not supported.

Why do you need to close a project?

**Comment by dang1507** [comment]
Hi Marcus,

I just finished my the capstone of my degree, which I tracked in projects.
It worked great for this.

Now I'm done. I didn't see amy way to mark a project as finished.

It was more of a question than a feature request. What should be done with
a finished project? Delete? Or just keep staring new ones?


Thanks,

Daniel

**Comment by Acylation** [comment, marked as answer]
Since [v1.17.0](https://github.com/marcusolsson/obsidian-projects/releases/tag/1.17.0), you can mark a project as `Archive` and there's a way to restore. Check #693 for more details!

Please note that for folder based archives, the `source path` settings of projects won't change after archiving or restoring. So you may need to pull back the files manually if you stored them elsewhere after archiving the project.
